apps/antalmanac/tests/patch-notes.test.tsx

```diff
@@ -0,0 +1,79 @@
+import { describe, expect, it } from 'vitest';
+import { render, screen, fireEvent } from '@testing-library/react';
+import PatchNotes from '$components/PatchNotes';
+
+const setLocalStorage = (key: string, value: string) => {
+    window.localStorage.setItem(key, value);
+};
+
+describe('PatchNotes Component', () => {
```

Comment:
I think this description is fine.
```suggestion
describe('patch notes', () => {
```